scripts/smoke-test-android.ps1

```diff
@@ -24,7 +24,7 @@ Write-Host "#####################################################"
 
 $BuildDir = $(GetNewProjectBuildPath)
 $ApkFileName = "test.apk"
-$ProcessName = "com.DefaultCompany.$(GetNewProjectName)"
+$ProcessName = "io.sentry.unity.integrationtest"
```

Comment:
With Unity 6 integration tests seem to fail to have their proper identifier set, causing the build to fail. We're setting this now explicitly in the `Builder.cs` and making use of the hardcoded name here.